teuthology/task/ceph_ansible.py

```diff
@@ -212,6 +212,34 @@ def teardown(self):
         os.remove(self.inventory)
         os.remove(self.playbook_file)
         os.remove(self.extra_vars_file)
+        # run purge-cluster that teardowns the cluster
+        args = [
+            'ANSIBLE_STDOUT_CALLBACK=debug',
+            'ansible-playbook', '-vv',
+            '-e', 'ireallymeanit=yes',
+            '-i', 'inven.yml', 'infrastructure-playbooks/purge-cluster.yml'
+        ]
+        log.debug("Running %s", args)
+        str_args = ' '.join(args)
+        installer_node = self.ceph_installer
+        if self.config.get('rhbuild'):
+            installer_node.run(
+                            args=[
```

Comment:
Could you remove some of the whitespace here on the left please? Something more like:

```
        if self.config.get('rhbuild'):
             installer_node.run(
                 args=[
                     run.Raw('cd ~/ceph-ansible'),
                     run.Raw(';'),
                     run.Raw(str_args)
                 ]
             )
         else:
```